sdks/python/apache_beam/transforms/fully_qualified_named_transform.py

```diff
@@ -52,16 +52,37 @@ def __init__(self, constructor, args, kwargs):
     self._kwargs = kwargs
 
   def expand(self, pinput):
-    return pinput | self._resolve(self._constructor)(
-        *self._args, **self._kwargs)
+    if self._constructor in ('__callable__', '__constructor__'):
```

Comment:
So the API is that user has to specific one of these strings as the method name and also make sure that the filter allows the same string ?